src/test/java/net/sf/jabref/exporter/MsBibExportFormatTest.java

```diff
@@ -0,0 +1,58 @@
+package net.sf.jabref.exporter;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.charset.Charset;
+import java.util.Collections;
+import java.util.List;
+
+import net.sf.jabref.BibDatabaseContext;
+import net.sf.jabref.Globals;
+import net.sf.jabref.JabRefPreferences;
+import net.sf.jabref.MetaData;
+import net.sf.jabref.model.database.BibDatabase;
+import net.sf.jabref.model.entry.BibEntry;
+
+import com.google.common.base.Charsets;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+import static org.junit.Assert.assertEquals;
+
+public class MsBibExportFormatTest {
+
+    public static BibDatabaseContext databaseContext;
+    public static Charset charset;
+    public List<BibEntry> entries;
+    public File tempFile;
+    public static MSBibExportFormat msBibExportFormat;
+
+    @Rule
+    public TemporaryFolder testFolder = new TemporaryFolder();
+
+
+    @Before
+    public void setUp() throws Exception {
+        Globals.prefs = JabRefPreferences.getInstance();
+        databaseContext = new BibDatabaseContext(new BibDatabase(), new MetaData());
+        charset = Charsets.UTF_8;
+        msBibExportFormat = new MSBibExportFormat();
+    }
+
+    @Test
+    public final void testPerformExportWithNoEntry() throws IOException {
+        entries = Collections.EMPTY_LIST;
+        tempFile = testFolder.newFile();
+        String tempFileName = tempFile.getCanonicalPath();
+        msBibExportFormat.performExport(databaseContext, tempFileName, charset, entries);
```

Comment:
Can you please add a overload to the performExport method in the export ExportFormats which accepts the file as an nio.Path instead of a stringy pathname.